.github/workflows/dockerhub-publish.yml

```
@@ -0,0 +1,196 @@
+            isLatest: true
+
+          # Pycharm community
+          - imageName: projector-pycharm-c
+            tag: 2019.3.4
+            downloadLink: https://download.jetbrains.com/python/pycharm-community-2019.3.4.tar.gz
+          - imageName: projector-pycharm-c
+            tag: 2020.2
+            downloadLink: https://download.jetbrains.com/python/pycharm-community-2020.2.tar.gz
+          - imageName: projector-pycharm-c
+            tag: 2020.3.3
+            downloadLink: https://download.jetbrains.com/python/pycharm-community-2020.3.3.tar.gz
+          - imageName: projector-pycharm-c
+            tag: 2020.3.5
+            downloadLink: https://download.jetbrains.com/python/pycharm-community-2020.3.5.tar.gz
+            isLatest: true
+
+          # Pycharm professional
+          - imageName: projector-pycharm-p
+            tag: 2019.3.4
+            downloadLink: https://download.jetbrains.com/python/pycharm-professional-2019.3.4.tar.gz
+          - imageName: projector-pycharm-p
+            tag: 2020.2
+            downloadLink: https://download.jetbrains.com/python/pycharm-professional-2020.2.tar.gz
+          - imageName: projector-pycharm-p
+            tag: 2020.3.3
+            downloadLink: https://download.jetbrains.com/python/pycharm-professional-2020.3.3.tar.gz
+          - imageName: projector-pycharm-p
+            tag: 2020.3.5
+            downloadLink: https://download.jetbrains.com/python/pycharm-professional-2020.3.5.tar.gz
+            isLatest: true
+
+          # Rider
+          - imageName: projector-rider
+            tag: 2020.2.4
+            downloadLink: https://download.jetbrains.com/rider/JetBrains.Rider-2020.3.3.tar.gz
+          - imageName: projector-rider
+            tag: 2020.3.3
+            downloadLink: https://download.jetbrains.com/rider/JetBrains.Rider-2020.3.3.tar.gz
+          - imageName: projector-rider
+            tag: 2020.3.4
+            downloadLink: https://download.jetbrains.com/rider/JetBrains.Rider-2020.3.4.tar.gz
+            isLatest: true
+
+          # Rubymine
+          - imageName: projector-rubymine
+            tag: 2020.3.2
+            downloadLink: https://download.jetbrains.com/ruby/RubyMine-2020.3.2.tar.gz
+          - imageName: projector-rubymine
+            tag: 2020.3.4
+            downloadLink: https://download.jetbrains.com/ruby/RubyMine-2020.3.3.tar.gz
+            isLatest: true
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+        with:
+          path: projector-server
+
+      - name: Checkout docker repo
+        uses: actions/checkout@v2
+        with:
+          repository: JetBrains/projector-docker
+          path: projector-docker
+
+      - name: Login to DockerHub
+        uses: docker/login-action@v1
+        with:
+          username: ${{ secrets.DOCKER_USERNAME }}
+          password: ${{ secrets.DOCKER_PASSWORD }}
+
+      - name: Push ide
```

Well, let's rename it a bit to make it more meaningful.

```suggestion
      - name: Push exact version
```